common/app/cards/CardType.scala

```diff
@@ -4,6 +4,8 @@ sealed trait CardType {
   val cssClassName: String
 
   def showVideoPlayer = this match {
+      // TODO check this is really correct - only should be playable if they're big enough - Third could be playable on desktop, but at mobile it renders them wide, narrow, narrow above each other
+      // so only the top one makes sense to be playable
```

Comment:
ok thanks, as discussed with you and @crifmulholland I will show the play button at third size, but use CSS rules to hide them when they go down to tiny size at mobile breakpoint - I will do this in a new PR.